src/main/java/com/wcc/platform/service/CmsService.java

```diff
@@ -35,4 +37,18 @@ public TeamPage getTeam() {
             throw new PlatformInternalException(e.getMessage(), e);
         }
     }
+
+    /**
+     * API to retrieve Code of conduct.
```

Comment:
Sorry, my bad, this javadoc about API should be in the controller. 

For the service, we should talk what is doing here, for example: 

```suggestion
     * Read JSON and convert to Pojo CodeOfConductPage 
```

Reply:
@goelsonali @dricazenck I updated also for footer controller and service functions.